src/meshpy/mesh_creation_functions/nurbs_geometries.py

```diff
@@ -141,6 +141,101 @@ def create_nurbs_hollow_cylinder_segment_2d(
     return surf
 
 
+def create_nurbs_cylindrical_shell_sector(
+    radius: float, angle: float, length: float, *, n_ele_u: int = 1, n_ele_v: int = 1
+) -> _NURBS.Surface:
+    """Creates a NURBS surface representing a 3D sector of a cylindrical shell.
+    The center of the cylindrical shell sector is located at [0, 0, 0].
+
+    Args:
+        radius (float): Radius of the cylindrical shell.
+        angle (float): Angle of the cylindrical shell sector in radians.
+                       The angle is only valid for 0 < angle <= pi according
+                       to "Isogeometric Analysis: Toward Integration of CAD and FEA" by
+                       J.Austin Cottrell, 2009
+        length (float): Length of the cylindrical shell.
+        n_ele_u (int, optional): Number of elements in the parametric u-direction. Defaults to 1.
+        n_ele_v (int, optional): Number of elements in the parametric v-direction. Defaults to 1.
+
+    Returns:
+        _NURBS.Surface: A geomdl object that contains the surface information.
+    """
```

Comment:
```suggestion
    """Creates a NURBS surface representing a 3D sector of a cylindrical shell.
    The center of the cylindrical shell sector is located at [0, 0, 0].

    Args:
        radius: Radius of the cylindrical shell.
        angle: Angle of the cylindrical shell sector in radians.
                       The angle is only valid for 0 < angle <= pi according
                       to "Isogeometric Analysis: Toward Integration of CAD and FEA" by
                       J.Austin Cottrell, 2009
        length: Length of the cylindrical shell.
        n_ele_u: Number of elements in the parametric u-direction. Defaults to 1.
        n_ele_v: Number of elements in the parametric v-direction. Defaults to 1.

    Returns:
        geomdl.NURBS.Surface: A geomdl object that contains the surface information.
    """
```

No need to specify the types in the documentation if you already sue type hints in the function header.

Reply:
Thanks for the suggestion!